packages/docs/community/release-process/attestation-service.md

```diff
@@ -0,0 +1,143 @@
+The latest image qualified for deployment to various networks are tagged as follows:
+
+* Alfajores: `attestation-service-alfajores`
+* Baklava: `attestation-service-baklava`
+* Mainnet: `attestation-service-mainnet`
+
+## Build process
+
+### Docker images
+
+Docker images are built automatically with [Google Cloud Build](https://cloud.google.com/cloud-build) upon pushes to `master` and all release branches. Automated builds will be tagged in [Google Container Registry](https://cloud.google.com/container-registry) with the corresponding commit hash.
+
+A signature should be produced over the image automatically built at the corresponding commit hash and included with the Github release.
+
+Release image signatures can be verified with the following command:
+
+```bash
+docker save $(docker image inspect us.gcr.io/celo-testnet/celo-monorepo:attestation-service-vX.Y.Z -f '{{ .Id }}') | gpg --verify attestation-service-vX.Y.Z.docker.asc -
+```
+
+## Testing
+
+As well as monorepo CI tests, all releases are expected to go through manual testing as needed to verify security properties, accuracy of documentation, and compatibility with deployed and anticipated versions of `celocli` and wallets including Valora.
+
+This testing should include running the Valora e2e tests:
+
+```bash
+git checkout verification-e2e-tests
+cd packages/mobile
+yarn test:e2e:ios -t e2e/src/RedeemInviteAndVerify.spec.js -i
+```
+
+## Promotion process
+
+### Source control
+
+Patch releases should be constructed by cherry-picking all included commits from `master` to the `release/attestation-service/x.y` branch. The first commit of this process should change the version number encoded in the source from `x.y.z-stable` to `x.y.z+1-unstable` and the final commit should change the version number to `x.y.z+1-stable`.
+
+Major and minor releases should be constructed by pushing a commit to the `master` branch to change the encoded version number from `x.y.z-unstable` to `x.y.z`. A `release/attestation-service/x.y` branch should be created from this commit.  The next commit must change the version number from `x.y.z-stable` to `x.y+1.0-unstable`, or `x+1.0.0-unstable` if the next planned release is a major release.
+
+Only one commit should ever have a “stable” tag at any given version number. When that commit is created, a tag should be added along with release notes. Once the tag is published it should not be reused for any further release or changes.
+
+### Distribution
+
+Distribution of an image should occur along the following schedule:
+
+<table>
+  <tr>
+    <td>Date</td>
+    <td>Action</td>
+  </tr>
+  <tr>
+    <td>T-1w</td>
+    <td>
+      <ol>
+        <li>Deploy release candidate build to alfajoresstaging and alfajores testnets</li>
+      </ol>
+    </td>
+  </tr>  
+  <tr>
+    <td>T</td>
+    <td>
+      <ol>
+        <li>Confirm Valora production and testing builds against Alfajores experience no issues and that e2e verification tests complete successfully</li>
```

Comment:
Is there an indication as to how to run the e2e tests? Or how to get access to the Valora production app against Alfajores?

Reply:
see above